content/applications/sales/crm/pipeline/multi_sales_team.rst

```diff
@@ -2,24 +2,63 @@
 Multiple sales teams
 ====================
 
-Use the *Sales Teams* feature to manage several sales teams, departments, or channels, each with
-their own unique sales processes.
+The *Multi Teams* feature in Odoo's *CRM app* allows salespeople to be assigned to multiple sales
+teams. Multiple teams can be created and manage in one dashboard, each with their own pipeline for
+leads and opportunities.
 
+Enable multi teams
+==================
+
+To allow salespeople to be assigned to more than one sales team, the :guilabel:`Multi Teams` setting
+needs to be enabled. First, navigate to :menuselection:`CRM --> Configuration --> Settings`. Under
+the :guilabel:`CRM` section, tick the checkbox labeled :guilabel:`Multi Teams`. Then, click
+:guilabel:`Save` at the top-left of the page.
+
+.. image:: multi_sales_team/multi-team-setting.png
+   :align: center
+   :alt: The Multi Teams setting in the CRM app.
+
+.. _crm/assignment-rules:
+
+Enable assignment rules
+=======================
+
+To periodically :doc:`assign leads <../track_leads/lead_scoring>` based on rules, the
+:guilabel:`Rule-Based Assignment` feature needs to be enabled. Navigate to :menuselection:`CRM -->
+Configuration --> Settings`. Under the :guilabel:`CRM` section, tick the checkbox labeled
+:guilabel:`Rule-Based Assignment`.
+
+This causes a new field, :guilabel:`Running` to appear. Select an option from the drop-down to
+manage how assignments are triggered.
+
+The rule-based assignment feature can be set to run :guilabel:`Manually`, meaning an Odoo user must
+manually trigger the assignment, or :guilabel:`Repeatedly`, meaning Odoo automatically triggers the
+assignments.
+
+Selecting :guilabel:`Repeatedly` causes additional fields to appear. Use the :guilabel:`Repeat
+every` fields to select how frequently the rules should be triggered. The :guilabel:`Next Run` field
+automatically updates, thought it can be manually updated.
```

Comment:
```suggestion
automatically updates, though it can be manually updated, if needed.

Once all desired configurations are complete, click :guilabel:`Save`.
```